components/datetime/src/pattern/formatter.rs

```diff
@@ -161,13 +161,11 @@ where
     /// use writeable::assert_try_writeable_eq;
     ///
     /// let mut london_winter = IxdtfParser::new()
-    ///     .try_from_str("2024-01-01T00:00:00+00:00[Europe/London]")
-    ///     .unwrap()
-    ///     .to_calendar(Gregorian);
+    ///     .try_from_str("2024-01-01T00:00:00+00:00[Europe/London]", Gregorian)
```

Comment:
Issue: This string should fail to parse because the default calendar is Iso, not Gregorian. Right?

Reply:
It parses the date as ISO and converts to Gregorian.